build_library/build_image_util.sh

```diff
@@ -154,6 +154,15 @@ write_packages() {
     fi
 }
 
+extract_docs() {
+    local root_fs_dir="$1"
+
+    info "Extracting docs"
+    tar --create --bzip --file="${BUILD_DIR}/docs.tar" \
```

Comment:
Oh, and probably remove extract_docs for the dev image, otherwise building a dev and prod image at the same time create conflicting tarballs.